src/main/java/org/axion/axion_api_gateway/AxionApiGatewayApplication.java

```diff
@@ -29,6 +31,8 @@ public static void main(String[] args) {
                                 )))
                 .start(24001);
 
+        app.get("/api/current-session", AxionApiGatewayApplication::currentSession);
```

Comment:
New mapping for current user's session as we don't want to forward this request to any other microservice.